modules/tester.js

```diff
@@ -1063,7 +1063,7 @@ Tester.prototype.begin = function begin() {
         config = getConfig([].slice.call(arguments)),
         next = function() {
             config.test(this, this.casper);
-            if (this.options.concise)
+            if (!this.options.concise)
```

Comment:
Well, I can add braces, but this file has some other if statements without braces.
What is your current styleguide? Braces for every if statement?